src/relatedContent.test.ts

```diff
@@ -0,0 +1,337 @@
+import { ElementType } from '@guardian/content-api-models/v1/elementType';
+import { AssetType } from '@guardian/content-api-models/v1/assetType';
+import { Block } from '@guardian/content-api-models/v1/block';
+import { Blocks } from '@guardian/content-api-models/v1/blocks';
+import { Asset } from '@guardian/content-api-models/v1/asset';
+import { ImageElementFields } from '@guardian/content-api-models/v1/imageElementFields';
+import { AssetFields } from '@guardian/content-api-models/v1/assetFields';
+import { RelatedItemType } from '@guardian/apps-rendering-api-models/relatedItemType';
+import { TagType } from '@guardian/content-api-models/v1/tagType';
+import { BlockElement } from '@guardian/content-api-models/v1/blockElement';
+
+let defaultBlock: Block;
+let defaultBlocks: Blocks;
+let imageAsset: Asset;
+let elementImageTypeData: ImageElementFields;
+let imageAssetTypeData: AssetFields;
+let defaultContent: Content;
+let imageElement: BlockElement;
+
+const createDefaultContent = () => {
+	return {
+		id: 'contentId',
+		type: ContentType.ARTICLE,
+		webTitle: 'contentTitle',
+		webUrl: 'contentUrl',
+		apiUrl: 'apiUrl',
+		tags: [],
+		references: [],
+		isHosted: true,
+		pillarId: 'somePillarId',
+		pillarName: 'somePillarName',
+	};
+};
+
+describe('parseRelatedContent', () => {
+	beforeEach(() => {
+		defaultContent = createDefaultContent();
+	});
+
+	it('returns related content', () => {
+		const actual = parseRelatedContent([defaultContent]);
+
+		const expected: RelatedContent = {
+			title: 'Related Stories',
+			relatedItems: [
+				{
+					title: 'contentTitle',
+					lastModified: undefined,
+					headerImage: undefined,
+					link: '/contentId',
+					type: RelatedItemType.ARTICLE,
+					pillar: {
+						id: 'somePillarId',
+						name: 'somePillarName',
+						sectionIds: [],
+					},
+					starRating: undefined,
+				},
+			],
+		};
+
+		expect(actual).toEqual(expected);
+	});
+
+	it('returns default pillar id given undefined content pillar id', () => {
+		defaultContent.pillarId = undefined;
+		const actual = parseRelatedContent([defaultContent]);
+		expect(actual.relatedItems[0].pillar.id).toEqual('pillar/news');
+	});
+
+	it('returns default pillar name given undefined content pillar name', () => {
+		defaultContent.pillarName = undefined;
+		const actual = parseRelatedContent([defaultContent]);
+		expect(actual.relatedItems[0].pillar.name).toEqual('news');
+	});
+
+	it('returns lastModified DateTime in response given content has lastModified field', () => {
+		const capiDateTime: CapiDateTime = {
+			dateTime: new Int64(6545664),
+			iso8601: 'iso',
+		};
+		const contentFields: ContentFields = {
+			lastModified: capiDateTime,
+		};
+		defaultContent.fields = contentFields;
+
+		const actual = parseRelatedContent([defaultContent]);
+
+		expect(actual.relatedItems[0].lastModified).toEqual(capiDateTime);
+	});
+
+	it('returns the first 4 related content given more than 4 content items', () => {
```

Comment:
This test doesn't check if the 4 items are actually the first 4, it only checks that the it returns 4 items.